### ISSUE TITLE
Issue with buttons appearing for Stash 3.2.0

### ISSUE THREAD

**Issue by mattadamson** [tomasbjerre/pull-request-notifier-for-bitbucket#158]
Team,

We're using Stash 3.2.0 which installed 1.29 of the plug in and can't seem to find any custom buttons in the PR view. We have configured PR events such as OPENED / RESCOPED too. 

Could this simply be a bug in an older version and we can't upgrade due to the old version of Stash we're using?

Thanks

**Comment by tomasbjerre** [comment]
It should work. What is the config that you apply? A screenshot of the config GUI would be nice.

**Comment by mattadamson** [comment]
actually just before I sent screenshots I tried updating to indicate every single PR event not just OPENED / RESCOPED and the build button appeared.  I wonder if it's also related to the UI save method e.g. the save buttons in each area. I noticed sometimes I had to remove extra configurations that were added to leave one.

**Comment by mattadamson** [comment]
I can open up another issue if you like however I noticed when I configured the URL as per the example blog

http://build.us.root.com:8080/job/PullRequestBuilderTest/buildWithParameters?${EVERYTHING_URL}

It failed and then noticed a stash exception in the log as jenkins returned a 500 response code. This is because Stash is sending the URL exact as that i.e. I was expecting the plug in to expand ${EVERYTHING_URL} to the various parameters exposed. 

Perhaps this is a feature only on a more recent version?

**Comment by tomasbjerre** [comment]
yes, EVERYTHING_URL is only available in recent versions. The button will only be visible if a notification is triggered by it.
